pom.xml

```diff
@@ -205,6 +229,21 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.eclipse.sisu</groupId>
+        <artifactId>sisu-maven-plugin</artifactId>
+        <version>0.9.0.M2</version>
```

Comment:
Already defined in parent ${sisuMavenPluginVersion}

Reply:
The upgrade to SISU has been done before the parent upgrade...
Removed in https://github.com/codehaus-plexus/plexus-compiler/pull/314/commits/0ba274be5bababfffb1f081e8a9c695694eaef4c